### ISSUE TITLE
Update color(grey-40)

### ISSUE THREAD

**Issue by joebell93** [sky-uk/toolkit-core#166]
>We need to change one of the greys in the colour palette as it turns out it fails AA accessibility for normal size text.
>We need to change **#7f7f7f** to **#707070**
– @danieljbryson 

As this is a breaking change we may have to consider it for Toolkit 2.0 release

**Comment by StefanMcCready** [comment]
Yeah I would say this would say this is for sure a 2.0 change

**Comment by joebell93** [comment]
Moved to https://github.com/sky-uk/toolkit/issues/185
